components/steering_wheel/include/IO_Digital.h

```diff
@@ -0,0 +1,36 @@
+/**
+ *IO_Digital.h
+ *Digital IO Module Header
+ */
+
+#pragma once
+
+/******************************************************************************
+ *                             I N C L U D E S
+ ******************************************************************************/
+
+#include <stdbool.h>
+
+/******************************************************************************
+ *                             T Y P E D E F S
+ ******************************************************************************/
+
+typedef struct
+{
+    struct
+    {
+        bool switch1:1;
+        bool switch2:1;
+        bool switch4:1;
+        bool switch5:1;
+        bool button_top_left:1;
+        bool button_middle:1;
+        bool button_top_right:1;
+    } input;
+} io_digital_S;
```

Comment:
the global struct should be all-caps, the local one in the C file should be lowercase. so this should be `IO_Digital_S`